haskell-hoogle.el

```diff
@@ -94,6 +96,18 @@ is asked to show extra info for the items matching QUERY.."
               "hoogle" "server" "-p" (number-to-string haskell-hoogle-port-number))))
     (error "\"hoogle\" executable not found")))
 
+(defun haskell-hoogle-start-stack-server ()
+  "Start hoogle local server using the local Stack configuration."
+  (interactive)
+  (if (executable-find "stack")
```

Comment:
I don't think you need this check, if the executable doesn't exist the user will get a suitable error anyway.